com.github.birros.WebArchives.json

```diff
@@ -0,0 +1,275 @@
+{
+    "id": "com.github.birros.WebArchives",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "web-archives",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=x11",
```

Comment:
you should also put --share=ipc when you expose --socket=x11. It improves performance.

Reply:
Ok, i'll add it.